backend/dataall/modules/redshift_datasets/api/datasets/types.py

```diff
@@ -70,6 +70,8 @@
             type=gql.Ref('RedshiftConnection'),
             resolver=resolve_dataset_connection,
         ),
+        gql.Field('successTables', gql.ArrayType(gql.String)),
```

Comment:
would this cause any issues for other queries/mutations that return `RedshiftDataset` but do not have `successTables` or `errorTables` defined (such as `updateRedshiftDataset` or `getRedshiftDataset`)?

Reply:
No, because we are not trying to fetch that data in the definition of those API calls

Reply:
but programmatically we could/would be with SDK/CLI